pixart.js

```diff
@@ -1 +1,45 @@
+$(document).ready(function(){
+  var textField = $('input:text');
+  var button = $("#set-color");
+  var brush = $(".brush")
 
+  function changeBrushColorOnClick() {
+    button.on("click", function(evt){
+      evt.preventDefault()
+      var colorInput = text.val();
+      brush.css("background", colorInput);
+    })
+  }
```

Comment:
functions like this that strictly just add event functionality don't even need to be abstracted into a function. We can just add the event listener. The only reason we would abstract this is if we knew going forward we needed to reattach this click event more than once.